exercises/anagram/src/test/java/AnagramTest.java

```diff
@@ -35,10 +43,17 @@ public void testDetectMultipleAnagrams() {
 
     @Ignore
     @Test
+    public void testDetectMultipleAnagramsForLongerWord() {
+        Anagram detector = new Anagram("allergy");
+        List<String> anagrams = detector.match(Arrays.asList("gallery", "ballerina", "regally", "clergy", "largely", "leading"));
+        assertThat(anagrams, allOf(hasItem("gallery"), hasItem("largely"), hasItem("regally")));
+    }
+
+    @Ignore
+    @Test
     public void testDoesNotConfuseDifferentDuplicates() {
         Anagram detector = new Anagram("galea");
-        List<String> anagrams = detector.match(Arrays.asList("eagle"));
-        assertTrue(anagrams.isEmpty());
+        assertTrue(detector.match(Arrays.asList("eagle")).isEmpty());
```

Comment:
I think it makes sense to group the two multiple anagrams tests together but I don't mind too much.